src/StateSmithTest/PlantUMLTests/ParsingTests.cs

```diff
@@ -473,6 +473,25 @@ @startuml blinky1_printf_sm
             """);
     }
 
+    /// <summary>
+    /// See https://github.com/StateSmith/StateSmith/issues/216
+    /// Ignore title line
+    /// </summary>
+    [Fact]
+    public void DiagramIgnoreTitle()
+    {
+        ParseAssertNoError("""
+            @startuml MyPumlSm1
+            title Test Title
+            @enduml
+            """);
+
+        ParseAssertNoError("""
+            @startuml MyPumlSm1
+            title ""Test Title""
+            @enduml
+            """);
+    }
```

Comment:
This looks good! 🎉 Is it possible to omit the other white space changes in this file though?